vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -561,7 +561,7 @@ def supports_cutlass_24(
 
         return weight_quant.num_bits == input_quant.num_bits == 8
 
-
+import vllm.envs as envs
```

Comment:
E402 Module level import not at top of file

Reply:
Updated